src/domUpdates.js

```diff
@@ -0,0 +1,172 @@
+            </button>
+            <label for='favorite-button' class='hidden'>Click to favorite recipe</label>
+            <button data-id='${recipe.id}' aria-label='favorite-button' class='favorite${recipe.id} favorite card-button'></button>
+          </header>
+            <span data-id='${recipe.id}' class='recipe-name'>${recipe.name}</span>
+            <img data-id='${recipe.id}' tabindex='0' class='card-picture'
+            src='${recipe.image}' alt='click to view recipe for ${recipe.name}'>
+      </div>`)
+    })
+  },
+
+  checkFavoriteActive(user) {
+    if (!user.favoriteRecipes.length) {
+      return
+    } else {
+      user.favoriteRecipes.forEach(recipe => {
+        $(`.favorite${recipe.id}`).addClass('favorite-active')
+      })
+    }
+  },
+
+  checkRecipeToCookActive(user) {
+    if (!user.recipesToCook.length) {
+      return
+    } else {
+      user.recipesToCook.forEach(recipe => {
+        $(`.to-cook${recipe.id}`).addClass('to-cook-active');
+      })
+    }
+  },
+
+  viewRecipesToCook(user, cookbook) {
+    $('.all-cards').removeClass('all')
+    if (!user.recipesToCook.length) {
+      $('.view-recipe-to-cook').html('No recipes saved!');
+      this.populateCards(cookbook.recipes, user);
+      return
+    } else {
+      $('.all-cards').html('');
+      $('.view-recipe-to-cook').html('Refresh Recipes to Cook');
+      this.createRecipeCards(user.recipesToCook);
+    }
+    this.checkRecipeToCookActive(user);
+  },
+
+  favoriteCard(event, cookbook, user) {
+    let specificRecipe = cookbook.recipes.find(recipe => {
+      if (recipe.id  === Number(event.target.dataset.id)) {
+        return recipe;
+      }
+    })
+    if (!$(event.target).hasClass('favorite-active')) {
+      $(event.target).addClass('favorite-active');
+      $('.view-favorites').html('View Favorites');
+      user.addToFavorites(specificRecipe);
+    } else if ($(event.target).hasClass('favorite-active')) {
+      $(event.target).removeClass('favorite-active');
+      user.removeFromFavorites(specificRecipe)
+    }
+  },
+
+  recipeToCookCard(event, cookbook, user) {
+    let specificRecipe = cookbook.recipes.find(recipe => {
+      if (recipe.id  === Number(event.target.dataset.id)) {
+        return recipe;
+      }
+    })
+    if (!$(event.target).hasClass('to-cook-active')) {
+      $(event.target).addClass('to-cook-active');
+      $('.view-recipe-to-cook').html('View Recipes to Cook');
+      user.addToRecipeToCook(specificRecipe);
+    } else if ($(event.target).hasClass('to-cook-active')) {
+      $(event.target).removeClass('to-cook-active');
+      user.removeFromRecipeToCook(specificRecipe);
+    }
+  },
+
+  displayDirections(event, cookbook, pantry, ingredientData, recipeData, user) {
+
+    let newRecipeInfo = cookbook.recipes.find(recipe => {
+      if (recipe.id === Number(event.target.dataset.id)) {
+        return recipe;
+      }
+    })
+    recipeObject = new Recipe(newRecipeInfo, ingredientData, recipeData);
```

Comment:
We can rename this since we don't want to name variables with the key name of Object